spec/system/job_seekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -60,22 +60,21 @@
         expect(page).to have_content(I18n.t('subscriptions.confirmation.header'))
       end
 
-      scenario 'when no reference is set' do
-        visit new_subscription_path(search_criteria: { keyword: 'test' })
-        fill_in 'subscription[email]', with: 'jane.doe@example.com'
-        click_on 'Subscribe'
+      context 'and is redirected to the confirmation page' do
```

Comment:
the context should be `when alert frequency is daily` and `when alert frequency is weekly`, and the scenarios `redirects to the confirmation page`

Reply:
https://www.betterspecs.org/